docs/ReleaseHowTo.md

```diff
@@ -0,0 +1,42 @@
+### How to release a new version of harvester CSI driver
+
+#### 1. Tag harvester CSI driver if needed. Sometimes, we could skip this part if only the support version was update on Rancher
+
+#### 2. Bump new version for Rancher apps
```

Comment:
Would it be worthwhile to add a little bit more context here for brand-new Harvester devs?

> harvester-csi-driver is distributed as a [standalone Helm package](https://github.com/harvester/charts/tree/master/charts/harvester-csi-driver) and is included in [Rancher's App catalogue](https://github.com/harvester/charts/tree/master/charts/harvester-csi-driver).

Reply:
Nice suggestion, updated!